clang/lib/Driver/Driver.cpp

```diff
@@ -8978,7 +9110,8 @@ InputInfoList Driver::BuildJobsForActionNoCache(
     if (isa<OffloadWrapperJobAction>(JA)) {
       if (Arg *FinalOutput = C.getArgs().getLastArg(options::OPT_o))
         BaseInput = FinalOutput->getValue();
-      else
+      // Do not use the default image name when using -fno-sycl-rdc
```

Comment:
Can you add an explanation in the comment of why we need -wrapper appended in this case?